# Descriptionâ€¦

geometry_msgs/Point base
geometry_msgs/Vector3 axis
float32 diameter		# m
float32 length			# m
